diff --git a/node_modules/react-native-css-interop/dist/metro/index.js b/node_modules/react-native-css-interop/dist/metro/index.js
index c91ba6a..9e37f6a 100644
--- a/node_modules/react-native-css-interop/dist/metro/index.js
+++ b/node_modules/react-native-css-interop/dist/metro/index.js
@@ -34,11 +34,6 @@ function getConfig(config, options) {
     const originalMiddleware = config.server?.enhanceMiddleware;
     const poisonPillPath = "./interop-poison.pill";
     fs_1.default.mkdirSync(outputDirectory, { recursive: true });
-    fs_1.default.writeFileSync(platformPath("ios"), "");
-    fs_1.default.writeFileSync(platformPath("android"), "");
-    fs_1.default.writeFileSync(platformPath("native"), "");
-    fs_1.default.writeFileSync(platformPath("macos"), "");
-    fs_1.default.writeFileSync(platformPath("windows"), "");
     return {
         ...config,
         transformerPath: require.resolve("./transformer"),
